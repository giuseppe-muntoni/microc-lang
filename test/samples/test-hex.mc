void main() {
    print(0x0);
    print(0x200);
    print(0xA);
    print(0xa);
    print(0xE);
    print(0xFFF);
}