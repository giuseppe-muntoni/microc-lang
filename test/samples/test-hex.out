0
512
10
10
14
4095
